### PR TITLE
NOTES.UNIX: add additional note about --enable-new-dtags

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

File: NOTES.UNIX [hunk 1]
```diff
@@ -28,3 +28,12 @@
 
     $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
         -Wl,-rpath,'$(LIBRPATH)'
+
+ On most GNU systems, a better choice is to use the new dtags, like this:
```
Comment (t-j-h) [inline NOTES.UNIX]: I don't think "GNU" is the right thing here ...

Comment (levitte) [reply NOTES.UNIX]: What is, then?  This seems to be very much tied in to systems that use the GNU toolchain all the way down to ld.so...  or is this Linux only?  Please advise

Comment (levitte) [reply NOTES.UNIX]: If [this wikipedia page](https://en.wikipedia.org/wiki/Rpath) is trustworthy, it does seem that "GNU" is the right thing.

Comment (levitte) [reply NOTES.UNIX]: Maybe it's better to start the sentence with "With GNU ld.so, a better choice ..."?

Comment (t-j-h) [reply NOTES.UNIX]: It is the "GNU system" I think is what was triggering the comment. It is about the combination of the compile time linker and runtime linker.

File: NOTES.UNIX [hunk 2]
```diff
@@ -28,3 +28,12 @@
...
+
+    $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
+        -Wl,--enable-new-dtags,-rpath,'$(LIBRPATH)'
+
+ This sets DT_RUNPATH instead of DT_RPATH.  DT_RUNPATH is considered after
+ the environment variable LD_LIBRARY_PATH, while DT_RPATH is considered
+ before that environment variable (making it useless).
```
Comment (t-j-h) [inline NOTES.UNIX]: It isn't "useless" - it is just different to what some people want. There are places for both behaviours.

Comment (levitte) [reply NOTES.UNIX]: Ah, I need to be more precise.  "it" refers to the environment variable `LD_LIBRARY_PATH`, not to what different people might desire.

**Comment by levitte** [comment]
Reworded.  Does it look better now?

**Review by t-j-h — APPROVED** [review]
Thanks for updating ...

**Comment by levitte** [comment]
Thank you!

Merged.  c3d76bb254eb33dbe2cd23e00f5f135f3c12381c

**Pull request closed without merging**